### PR TITLE
SC-5612 - Search bar for admintables

### PR DESCRIPTION
**Pull request by murat-merdoglu-hpi**
# Description

## Links to Tickets or other pull requests
https://ticketsystem.schul-cloud.org/browse/SC-5612

## Changes

- Added a searchbar component at the top of the students/teachers pages

- This feature helps the admin users to search students/teachers by using the search bar.
- It searches the "firstName, lastName and email" columns and filters.

## Approval for review

- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definiton of Done

More and detailed information on the _definition of done_ can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: locale/de.json [hunk 1]
```diff
@@ -264,6 +264,8 @@
 	"pages.administration.teachers.new.error": "Lehrer:in konnte nicht erfolgreich hinzugefügt werden. Möglicherweise existiert die E-Mail-Adresse bereits.",
 	"pages.administration.teachers.new.success": "Lehrer:in erfolgreich angelegt!",
 	"pages.administration.teachers.new.title": "Lehrer:in hinzufügen",
+	"pages.administration.teachers.index.searchbar.placeholder": "Suchen",
+	"pages.administration.students.index.searchbar.placeholder": "Suchen",
```
Comment (ABialoskorski) [inline locale/de.json:268]: please use only one key, delete duplicates

File: src/pages/administration/students/index.vue [hunk 2]
```diff
@@ -11,11 +11,19 @@
 			:backend-filtering="true"
 			:active-filters.sync="currentFilterQuery"
 		/>
+
+		<search-bar
+			v-model="searchQuery"
+			placeholder="Search"
```
Comment (ABialoskorski) [inline src/pages/administration/students/index.vue:17]: use i18n for placeholder

File: locale/de.json (hunk 1 above)
Comment (murat-merdoglu-hpi) [reply locale/de.json]: Corrected

File: src/pages/administration/students/index.vue (hunk 2 above)
Comment (murat-merdoglu-hpi) [reply src/pages/administration/students/index.vue]: Corrected

**Review by j-jmnz — APPROVED** [review]

**Review by j-jmnz — APPROVED** [review]

**Pull request merged**